src/main/java/seedu/address/logic/AnimalMessages.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.logic;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import seedu.address.logic.parser.Prefix;
+import seedu.address.model.animal.Animal;
+
+/**
+ * Container for user visible messages.
+ */
+public class AnimalMessages {
+
+    public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
+    public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_INVALID_ANIMAL_DISPLAYED_INDEX = "The animal index provided is invalid";
+    public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d animals listed!";
```

Comment:
As mentioned in an earlier PR, I do not see the rationale in specifying the arguments in the format string as such, since these strings only accept one argument. I believe it would be better to change it to `\n%s` and `%d` respectively.